docs(vault) - update sections for vault superflags (#133)
dgraph-io/dgraph-docs: content/enterprise-features/encryption-at-rest.md

If the Alpha server restarts, the `--encryption_key_file` option must be set along with the key in order to
restart successfully.

### Storing encryption key secret in Hashicorp Vault

The encryption key secret can be saved in [Hashicorp Vault](https://www.vaultproject.io/) K/V Secret instead of as file on Dgraph Alpha.

> To use [Hashicorp Vault](https://www.vaultproject.io/), these prerequisites must be met for the Vault Server.

Reviewer: ```suggestion
To use [Hashicorp Vault](https://www.vaultproject.io/), meet the following prerequisites for the Vault Server.
```

1. Accessible from Dgraph Alpha and configured using URL `http://fqdn[ip]:port`.
2. Enable [AppRole Auth method](https://www.vaultproject.io/docs/auth/approle) and enable [KV Secrets Engine](https://www.vaultproject.io/docs/secrets/kv).
3. Save the value of the key (16, 24, or 32 bytes long) that Dgraph Alpha will use in a KV Secret path (KV V1 or KV V2).  For example, you can upload this below to KV Secrets V2 path of `secret/data/dgraph/alpha`:
   ```json
   {
     "options": {
       "cas": 0
     },
     "data": {
       "enc_key": "1234567890123456"
     }
   }
   ```   
4. A role with an attached policy that grants access to the secret.  For example, this policy would grant access to `secret/data/dgraph/alpha`:
   ```hcl
   path "secret/data/dgraph/*" {
     capabilities = [ "read", "update" ]
   }
   ```

> 5. The `role-id` and corresponding `secret-id` must must be generated and copied over to local files.

Reviewer: ```suggestion
5. On the Vault server, generate the `role-id` and corresponding `secret-id`; and then save these ids on the Dgraph Alpha server. To learn more about this step, see [HashiCorp Vault Integration: Docker](https://github.com/dgraph-io/dgraph/blob/master/contrib/config/vault/docker/README.md).
```

Reviewer: Feel free to break this up into multiple steps, if you want, but if you do please specify for each action whether it takes place on the Vault or Dgraph Alpha server :)

> Here is an example of using Dgraph with a Vault server that holds the encryption key:

Reviewer: Here is an example > The following example

```bash
dgraph zero --my=localhost:5080 --replicas 1 --raft "idx=1"
dgraph alpha --my="localhost:7080" --zero="localhost:5080" \
  --vault addr="http://localhost:8200";enc-field="enc_key";enc-format="raw";path="secret/data/dgraph/alpha";role-id-file="./role_id";secret-id-file="./secret_id"
```

> If multiple Dgraph Alpha nodes are part of the cluster, you will need to pass the `--encryption_key_file` flag or the `--vault` superflag with appropriate superflag options to each of the Dgraph Alpha nodes.

Reviewer: will need to -> must
